docs/bundler/css.mdx

```diff
@@ -34,7 +34,7 @@ By default, Bun's CSS bundler targets the following browsers:
 
 The CSS Nesting specification allows you to write more concise and intuitive stylesheets by nesting selectors inside one another. Instead of repeating parent selectors across your CSS file, you can write child styles directly within their parent blocks.
 
-```css title="styles.css" icon="file-code"
```

Comment:
Why css → scss? Were these broken before?

Reply:
the ones with css nesting, i made scss because it makes shiki and most renders understand it better and actually highlight